src/rpc/rawtransaction_util.cpp

```diff
@@ -280,102 +274,101 @@ CMutableTransaction ConstructTransaction(const UniValue& inputs_in, const UniVal
 
     }
 
-    if (!outputs_is_obj) {
-        // Translate array of key-value pairs into dict
-        UniValue outputs_dict = UniValue(UniValue::VOBJ);
-        for (size_t i = 0; i < outputs.size(); ++i) {
-            const UniValue& output = outputs[i];
-            if (!output.isObject()) {
-                throw JSONRPCError(RPC_INVALID_PARAMETER, "Invalid parameter, key-value pair not an object as expected");
-            }
-            if (output.size() != 1) {
-                throw JSONRPCError(RPC_INVALID_PARAMETER, "Invalid parameter, key-value pair must contain exactly one key");
-            }
-            outputs_dict.pushKVs(output);
-        }
-        outputs = std::move(outputs_dict);
-    }
```

Comment:
Has this been done in Core as well? It feels weird to have a breaking RPC change in `createrawtransaction` for Elements 0.21 that isn't reflected in Core 0.21. I think we sholud revert this for 0.21.

Reply:
No, but because we need this extra data for elements, it gets really messy to maintain the other way too. The other way to specify outputs is supposed to be deprecated anyways, but it hasn't been removed.

Reply:
Ok, fair enough